test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -183,7 +186,8 @@ TEST(mongoRegisterContext_update, updateCase1)
   prepareDatabase();
 
   /* Invoke the function in mongoBackend library */
-  ms = mongoRegisterContext(&req, &res);
+  ms = mongoRegisterContext(&req, &res, "", "/");
+  LM_M(("Invoke mongoRegisterContext in mongoBackend library and EXIT. ms: %d", ms));
```

Comment:
We don't use to have LM_M() traces in unittest code... it is actually needed or a leftover?